docs/streams/developer-guide/config-streams.html

```diff
@@ -70,11 +70,10 @@
           <li><a class="reference internal" href="#optional-configuration-parameters" id="id6">Optional configuration parameters</a>
             <ul>
               <li><a class="reference internal" href="#acceptable-recovery-lag" id="id27">acceptable.recovery.lag</a></li>
-              <li><a class="reference internal" href="#default-deserialization-exception-handler" id="id7">default.deserialization.exception.handler</a></li>
```

Comment:
I think this one was on the correct place alphabetically? Not sue why you moved it?

Cf my other comments below (not sure how to structure this best)

Reply:
it was a mistake, sorted based on the href link which was incorrect. fixed it nw.